### PR TITLE
Fixes storybook and distributes stories for components folders

### PR DESCRIPTION
**Pull request by jayrmotta**
**Problem**: Storybook was not working.

**Solution**: For some reason storybook configured file path pattern was considering test files and therefore triggering an error trying to load jest, so I took it out and reorganized the stories in a more decentralised manner.

**Next steps**:
So, guys I think this is the first step towards building a more productive frontend code base to work since storybook will allow and encourage more reusability of the components we build.

I think the next steps are improving the directory structure as @almeynman already proposed, from that we could extract all the shared components in a way that the nexts screens we build are more like composing components than actually building from scratch and repeating code.

### CODE REVIEW

**Comment by almeynman** [comment]
Great stuff!

One thing though. There are many files named `...story.spec.js` which currently do not get pulled by storybook. Need to rename those to `story.js`

**Comment by v3rm0n** [comment]
If you rename them to `story.js` then they will not be picked up when tests are run.

**Comment by v3rm0n** [comment]
@jayrmotta maybe this will suffice? https://github.com/ChangeFinance/wallet-client/pull/193

**Comment by almeynman** [comment]
My current understanding is:
- stories apply only for presentational components
- for presentational components we combine story and spec in one file and it should be called `.story.spec.js`
- jest picks up files ending with `.spec.js`, so it should also pick up tests in stories as well

File: .storybook/config.js [hunk 1]
```diff
@@ -62,10 +62,9 @@ function Provider({ story }) {
 }
 addDecorator(story => <Provider story={story()} />);
 
-const req = require.context('../src', true, /\.story\.spec\.js$/);
```
Comment (almeynman) [inline .storybook/config.js]: This line should be reverted back with a fix from https://github.com/ChangeFinance/wallet-client/pull/193 to make storybook work

**Comment by jayrmotta** [comment]
@v3rm0n It does. Do you think excluding stories from coverage and spliting stories in many files is something we should do?

**Comment by jayrmotta** [comment]
@almeynman why do you think that stories and specs should be in the same file?

**Comment by jayrmotta** [comment]
OK, that's fine.

**Pull request closed without merging**